src/transformation/visitors/call.ts

```diff
@@ -196,12 +199,16 @@ function transformPropertyCall(
     }
 }
 
-function transformElementCall(context: TransformationContext, node: ts.CallExpression): lua.Expression {
+function transformElementCall(
+    context: TransformationContext,
+    node: ts.CallExpression,
+    customName?: string
```

Comment:
This doesn't make sense to me, how can `a[]` have a customName?, if `a` has a custom name it will be handled by transformIdentifier. Please revert this